avro/src/codec.rs

```diff
@@ -91,7 +91,7 @@ impl Codec {
                 let mut encoded: Vec<u8> = vec![0; snap::raw::max_compress_len(stream.len())];
                 let compressed_size = snap::raw::Encoder::new()
                     .compress(&stream[..], &mut encoded[..])
-                    .map_err(Error::SnappyCompress)?;
+                    .map_err(Details::SnappyCompress)?;
```

Comment:
How does this work without the `.into()` call ?

Reply:
The try operator (question mark: ?) automatically calls into